src/backend/opencl/platform.cpp

```diff
@@ -673,22 +672,26 @@ void DeviceManager::markDeviceForInterop(const int device, const fg::Window* wHa
 #if defined(_WIN32) || defined(_MSC_VER)
                 CL_WGL_HDC_KHR, (cl_context_properties)wHandle->display(),
 #else
-                CL_GLX_DISPLAY_KHR, (cl_context_properties)wHandle->display(),
+                //CL_GLX_DISPLAY_KHR, (cl_context_properties)wHandle->display(),
 #endif
                 CL_CONTEXT_PLATFORM, (cl_context_properties)plat(),
```

Comment:
Well the patch I sent seems to make it work. I was wondering if it is a good idea.